### PR TITLE
Release 0.9.92

### PR DESCRIPTION
**Pull request by KGuillemot**
### Improvements
-  RSYSLOG TEMPLATE : complete reputation tags, clarify template, add impcap tags :
    - 'impcap' template simplified to simple json
    - added impcap fields for IPv6 (route_seg_left, frag_offset, frag_more, frag_id)
    - added impcap fields for HTTP (version, status_code, reason, method, request_uri, header_fields)
   - added reputation tags (location longitude/latitude, city, country, tags)
- LOGROTATE : Add PacketFilter log file rotation

### Fixes
 - UPGRADE : Remove openvpn_client & telegraf rc conf files to prevent replacing custom config

### CODE REVIEW

File: usr/local/etc/logrotate.d/vulture.conf [hunk 1]
```diff
@@ -116,5 +116,21 @@
 	sharedscripts  # execute script one time for all logs
 	postrotate
 	    /usr/sbin/service syslogd restart
+	endscriptt
```
Comment (frikilax) [inline usr/local/etc/logrotate.d/vulture.conf]:
```suggestion
	endscript
```

File: usr/local/etc/rsyslog.d/05-tpl-02-impcap.conf [hunk 2]
```diff
@@ -1,95 +1,107 @@
...
-    constant(value="\", \"HTTP_status_code\": ")  property(name="$!impcap!HTTP_status_code")
-    constant(value=", \"HTTP_content_type\": \"") property(name="$!impcap!HTTP_content_type")
-    constant(value="\", \"reputation\": \"")        property(name="$!iplocation!reputation")
-    constant(value="\", \"geoip\": \"")             property(name="$!iplocation!geoip")
-    constant(value="\", \"ctx_tags\": \"")          property(name="$!ctx_tags")
-    constant(value="\", \"DNS_queries\": ")         property(name="$!impcap!DNS_queries")
-    constant(value="\", \"darwin_id\": ")         property(name="$!mmdarwin!darwin_id")
-    constant(value="}")
+template(name="impcap" type="list" option.jsonf="on") {
+    property(name="$!impcap!timestamp" outname="time" format="jsonf" datatype="string" dateformat="date-rfc3339")
```
Comment (KGuillemot) [inline usr/local/etc/rsyslog.d/05-tpl-02-impcap.conf]: dateformat must not contain "date-"

Comment (KGuillemot) [inline usr/local/etc/rsyslog.d/05-tpl-02-impcap.conf]:
```suggestion
    property(name="$!impcap!timestamp" outname="time" format="jsonf" datatype="string" dateformat="rfc3339")
```

File: usr/local/etc/rsyslog.d/05-tpl-02-impcap.conf [hunk 3]
```diff
@@ -1,95 +1,107 @@
...
+    property(name="$!iplocation!dst_location!latitude" outname="ctx_dst_geoip.latitude" format="jsonf" datatype="number")
+    property(name="$!iplocation!dst_location!longitude" outname="ctx_dst_geoip.longitude" format="jsonf" datatype="number")
+    property(name="$!iplocation!src_reputation" outname="ctx_src_reputation" format="jsonf" datatype="string")
+    property(name="$!iplocation!dst_reputation" outname="ctx_dst_reputation" format="jsonf" datatype="string")
 }
 
 
 template(name="impcap_mongo" type="list") {
-    property(name="$!impcap!timestamp" outname="time")
+    property(name="$!impcap!timestamp" outname="time" dateformat="date-rfc3339")
```
Comment (KGuillemot) [inline usr/local/etc/rsyslog.d/05-tpl-02-impcap.conf]:
```suggestion
    property(name="$!impcap!timestamp" outname="time" dateformat="rfc3339")
```

**Pull request merged**